app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<footer>
```

Comment:
I should have put them in the layouts/application.html.erb file since they were reused between all the views-- but I definitely got better at refactoring using partials with MediaRanker!